evap/student/templates/student_vote_questionnaire_group.html

```diff
@@ -26,7 +26,7 @@
                             {{ field.label }}
                             <div class="warning-label mt-auto d-none">
                                 <span class="fas fa-exclamation-triangle"></span>
-                                {% trans "This answer will be filtered out. Please leave the textbox empty if you don't have any feedback for this particular question." %}
+                                {% trans "Your answer seems meaningless. Please leave the textbox empty if you don't have any feedback for this question." %}
```

Comment:
What about this? I'd also prefer to not mention the filtering.

Reply:
maybe "unhelpful"?
we could even keep only the second sentence. it still explains what the user should do and is shorter.

Reply:
A/B testing! :rofl: ... I'd like to hear more opinions...

Reply:
then i'd say we should at least make it "will probably be filtered out" - that's something i could live with.

Reply:
yeah, i like the "probably". i'd like to have a tooltip on it that says "using advanced tool-assisted filtering procedures".

Reply:
So we're talking about "This answer will probably be filtered out. You can simply leave the textbox empty" now, right? I like that.